:hammer: Added changes to checker.py

yipkhenlai opened the pull request:
**Description**
To prevent errors running script happened when the folder is already created.

Commit:
:hammer: Added changes to checker.py

codenamewei commented on scripts/checker.py:
```diff
@@ -13,7 +13,11 @@
+
+    if path == None:
+        os.mkdir(path)
+    else:
```
can remove the else loop

yipkhenlai replied:
@codenamewei ok

yipkhenlai committed:
Update checker.py

removed else loop

codenamewei reviewed (changes requested):
I don't seems to able to enter the if loop with your code
when the path do not exist

<img width="609" alt="Screenshot 2021-06-18 at 9 26 16 AM" src="https://user-images.githubusercontent.com/33477318/122491848-54c5b380-d017-11eb-9f46-02743daa2b54.png">

ken479 commented:
@codenamewei You are right, I will correct the mistake.

yipkhenlai committed:
Update checker.py

Thank you chia wei for your review. Added condition to check if the folder exists.
